entrypoint.sh

```diff
@@ -69,6 +82,11 @@ echo "Configuring"
 
 unset RUNNER_TOKEN
 
+# Saving the files in another directory for the possibility to mount them from the host next time
+if [ -d ${CONFIGURED_ACTIONS_RUNNER_FILES_DIR} ]; then
+  cp -p -r "/actions-runner/_diag" "/actions-runner/svc.sh" /actions-runner/.[^.]* ${CONFIGURED_ACTIONS_RUNNER_FILES_DIR}
```

Comment:
this doesnt seem inherently easy to understand, im not a fan of regex bash calls. it also seems different than line 56

Reply:
56 line just copy all files because it is is not really important what it copies. If the user will try to mess the directory it is his problem and you just cannot remove all such the possibilities.

87 line copies only the files that appears only after the registration:
* `svc.sh`
* `_diag`
* `.credentials`
* `.credentials_rsaparams`
* `.env`
* `.path`
* `.runner`

Considering the possible changes I thought about simplifying and not limiting to the certain hidden files but all

For the copying only hidden files see [this](https://superuser.com/questions/830562/how-to-copy-hidden-starting-with-a-dot-files-and-subdirectories-in-linux)